express-crud/server.ts

```diff
@@ -0,0 +1,122 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: { rejectUnauthorized: false },
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+    select *
+    from "grades";
+    `;
+    const result = await db.query(sql);
+    res.status(200).json(result.rows);
```

Comment:
Status 200 is the default so `status(200)` isn't needed. But it's also fine having it here to be explicit about what you are doing.